### DISCUSSION TITLE
v1.16.21版本使用外部编辑器选项添加编辑

### DISCUSSION THREAD

**Discussion by 1783cwf** [electerm/electerm#2281] [Q&A]
### 疑问
我看新版本添加了一个使用外部编辑器编辑 ,这个功能是指使用外部编辑器编辑远程的文件吗?
### 环境
我现在使用的是win10,本地编辑器是subimeText3,
### 尝试
- 首先我使用命令的形式 但是点击编辑的时候还是原先的内置编辑器
![image](https://user-images.githubusercontent.com/56056304/137053845-e5111bf6-3458-40fb-b972-b5e3d9d9b271.png)
并且在sftp中出现了两个编辑
![image](https://user-images.githubusercontent.com/56056304/137053998-d795b5c1-b0f6-4861-ad5d-bacfaa6925f8.png)
![image](https://user-images.githubusercontent.com/56056304/137054030-1e5812d7-6f52-44ae-8561-6db2a6fddc38.png)
点击第二个编辑 点击open无反应
切换到路径的形式也是同样的情况

**Comment by zxdong262** [comment]
目前只支持exe的名称，也就是一定要已经在系统路径里, 绝对路径还不行
